agent/api/task/task_test.go

```diff
@@ -376,17 +376,6 @@ func TestDockerHostConfigPortBinding(t *testing.T) {
 					if !reflect.DeepEqual(config.PortBindings, tc.expectedPortBinding) {
 						t.Error("Expected port bindings to be resolved, was: ", config.PortBindings)
 					}
-				} else {
```

Comment:
__NEW!__: Remove singular host port validation in `TestDockerHostConfigPortBinding` as ports should be validated before returning back from  `utils.GetHostPort()`